profiles/coreos/base/package.accept_keywords

```diff
@@ -31,7 +31,9 @@ dev-util/checkbashisms
 
 =net-libs/gnutls-3.7.1 ~amd64 ~arm64
 
-=net-misc/openssh-8.7_p1-r1 ~amd64 ~arm64
+=net-libs/libmicrohttpd-0.9.55 ~amd64 ~arm64
```

Comment:
We should remove this no ? It seems it has been dropped in this PR: https://github.com/flatcar-linux/coreos-overlay/pull/1408/files since we're about to ship `net-libs/libmicrohttpd-0.9.73`.

Reply:
Good catch, thanks.
Probably something went wrong during the rebase.
Fixed.